.github/workflows/docker-image.yml

```diff
@@ -53,4 +52,44 @@ jobs:
     - name: Logout of Amazon ECR
       if: always()
       run: docker logout ${{ steps.login-ecr.outputs.registry }}
+
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Deploy to elia_staging
+      uses: peterkimzz/aws-ssm-send-command@master
+      id: ssm
+      with:
+        aws-region: eu-central-1
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        instance-ids: ${{ secrets.INSTANCE_ID }}
+
+        working-directory: /home/ubuntu/docker-stack/
+        command: ls -la
+        command: echo $USER
+        
+        #Setup aws Credentials/Variables
+        command: export AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Please don't set it like that. Set it as a parameter for .sh command, so it won't be available on the machine.

Eg:
```sh
install.sh --var=${{secrets.VAR}}
```

Reply:
They need to be set up either way in .aws config file or as an env variable. Thus i added "unset" in the end to clear it out.

Reply:
What if the pipe failed? Then the variable will not be unset